specification/domainservices/resource-manager/Microsoft.AAD/2017-01-01/DomainServicesSwagger.json

```diff
@@ -49,7 +49,7 @@
         },
         "x-ms-examples": {
           "Get Operations": {
-            "$ref": "../examples/GetOperations.json"
+            "$ref": "/examples/GetOperations.json"
```

Comment:
I believe this references have to be "./examples..." so start with "." - https://travis-ci.org/Azure/azure-rest-api-specs/jobs/277044016